docs/get-started/getting-started-gcp.md

```diff
@@ -169,3 +172,91 @@ The AI platform will connect to a Dataproc cluster through a yaml configuration.
 
 In the future, users will be able to provision a Dataproc cluster through DataprocHub notebook.  You
 can use example [pyspark notebooks](../demo/GCP/Mortgage-ETL-GPU.ipynb) to experiment.
+
+## Build custom dataproc image to accelerate cluster init time
+In order to accelerate cluster init time to 4-5 minutes, we need to build a custom Dataproc image
+that already has NVIDIA drivers and CUDA toolkit installed. In this section, we will be using [these
+instructions from GCP](https://cloud.google.com/dataproc/docs/guides/dataproc-images) to create a
+custom image. 
+
+Currently, the [GPU Driver](https://github.com/GoogleCloudDataproc/initialization-actions/tree/master/gpu) initialization actions:
+1. Configure YARN, the YARN node manager, GPU isolation and GPU exclusive mode.
+2. Install GPU drivers.
+
+While step #1 is required at the time of cluster creation, step #2 can be done in advance. Let's
+write a script to do that. `gpu_dataproc_packages.sh` will be used to create the Dataproc image:
+
+<script src="https://gist.github.com/aroraakshit/191f4435c825f89f06f108691e104074.js"></script>
+
+Google provides a `generate_custom_image.py` script that:
+- Launches a temporary Compute Engine VM instance with the specified Dataproc base image.
+- Then runs the customization script inside the VM instance to install custom packages and/or update configurations. 
+- After the customization script finishes, it shuts down the VM instance and creates a Dataproc custom image from the disk of the VM instance.
+- The temporary VM is deleted after the custom image is created. 
+- The custom image is saved and can be used to create Dataproc clusters.
+
+Let's create the image using our customization script `gpu_dataproc_packages.sh` and image creation
+script `generate_custom_image.py` by Google (this step may take 20-25 minutes to complete):
+
+```bash
+git clone https://github.com/GoogleCloudDataproc/custom-images
+cd custom-images
+
+export CUSTOMIZATION_SCRIPT=/path/to/gpu_dataproc_packages.sh
+export ZONE=[Your Preferred GCP Zone]
```

Comment:
Why `ZONE` instead of `REGION` in this script?

Reply:
No particular reason, simply following the user guide here: https://github.com/GoogleCloudDataproc/custom-images#generate-custom-image